precompile/fee_config_manager.go

```diff
@@ -298,6 +300,7 @@ func StoreFeeConfig(stateDB StateDB, feeConfig commontype.FeeConfig, blockContex
 		case blockGasCostStepKey:
 			input = common.BigToHash(feeConfig.BlockGasCostStep)
 		default:
+			// this should not ever happen.
```

Comment:
can we make this comment match the others? fwiw I think we could get rid of these comments here since it should be obvious from the panic that this should not happen

Reply:
I'd prefer to keep this, as we're panicking in default case which is not ideal for a "default" :D